### PR TITLE
Doxygen 1.9.3 with https url and sha256 update

### PR DESCRIPTION
**Pull request by VincentRouvreau**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by MuellerSeb** [comment]
Looks good to me!

**Comment by VincentRouvreau** [comment]
> Looks good to me!

Thanks ! Let's see if @tadeu @isuruf or @bluescarni can take some time to review and merge it.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,16 +7,16 @@ package:
 
 source:
   fn: {{ name }}-{{ version }}.src.tar.gz
-  url: http://doxygen.nl/files/{{ name }}-{{ version }}.src.tar.gz
-  sha256: adcd968e10f27a15c956aaad86f544174f0e3d2d26d8d978929513ceeb09c19d
+  url: https://www.doxygen.nl/files/{{ name }}-{{ version }}.src.tar.gz
+  sha256: f352dbc3221af7012b7b00935f2dfdc9fb67a97d43287d2f6c81c50449d254e0
```
Comment (tadeu) [inline recipe/meta.yaml:11]: It's so weird that Doxygen's tarballs keep changing without a version change, I wonder if [GitHub releases](https://github.com/doxygen/doxygen/releases/tag/Release_1_9_3) would be more stable.

**Review by tadeu — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (VincentRouvreau) [reply recipe/meta.yaml:11]: @tadeu some other weird thing is that CI was successful on this PR but failed for windows CI once merged. There is an issue with initexmf:
```
Warning: Option --admin should be specified when running this program with administrative privileges
```
As it is installed by conda package miktex (version 2.9.6753-h2fa13f4_1000 on failed and success CI), I have no clue on how to fix it.

Comment (tadeu) [reply recipe/meta.yaml:11]: Thanks for noticing that, maybe @isuruf or @conda-forge/miktex can help with this?

Here are both builds, for comparison:

Failed on `main`: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=521030&view=logs&j=a70f640f-cc53-5cd3-6cdc-236a1aa90802&t=f5d15007-a01c-5ad8-c9ce-4d519d3b275f

Passed on PR: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=519869&view=logs&j=a70f640f-cc53-5cd3-6cdc-236a1aa90802&t=f5d15007-a01c-5ad8-c9ce-4d519d3b275f

**Pull request merged**